src/Settings.php

```diff
@@ -3,6 +3,7 @@
 namespace P4\MasterTheme;
 
 use CMB2_Field;
+use P4\MasterTheme\Settings\InformationArchitecture;
```

Comment:
In the [MasterSite](https://github.com/greenpeace/planet4-master-theme/pull/1591/files#diff-d5a71432d6e94abc1260a448df306d8dfc76d15fa4047646a4d133c38cc131d7R14) file you renamed this as `IA`, maybe it would be good to do that here too for consistency?